### DISCUSSION TITLE
How to seed database conditionally?

### DISCUSSION THREAD

**Discussion by sakhmedbayev** [prisma/prisma#13099] [Q&A]
Let's say I want to seed a database in production with certain seeds and in development with other seeds. How can I accomplish it? Can Prisma support the "latest" seeds, so I can always apply the latest seeds at the same time when I apply the latest migration?

**Comment by nurul3101** [comment]
Hey @sakhmedbayev 👋 

You could have different scripts which you could run against different environments.

You could also have a look at [snaplet.dev](https://www.snaplet.dev/) which might be useful in this use case.
Can you please elaborate on "latest" seeds point, I didn't understand it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
